terraform/frontend/interface/src/environments/environment.prod.ts

```diff
@@ -1,8 +1,8 @@
 export const environment = {
   production: true,
   stage: 'prod',
-  build: '1.0.1',
-  buildDateTime: '2025-03-21 14:45 EST',
+  build: '1.0.2',
+  buildDateTime: '2025-05-07 12:00 EST',
```

Comment:
@NathanielMartin42 after this PR is approved, but before you merge it, update the buildDateTime to date/time when you think you'll actually deploy to prod. I'm okay with committing this directly to dev branch.